drivers/soundwire/stream.c

```diff
@@ -1283,7 +1283,7 @@ static int sdw_slave_port_config(struct sdw_slave *slave,
 		 * TODO: Check valid port range as defined by DisCo/
 		 * slave
 		 */
-		ret = sdw_is_valid_port_range(&slave->dev, port_config[i].num);
+		ret = sdw_slave_port_is_valid_range(&slave->dev, port_config[i].num);
```

Comment:
we could do this as a follow-up optimization. I wanted to keep this as is since there's a TODO that should be implemented at some point with additional checks. For now we only check for the min and max ranges allowed in the standard for data ports, but if/when platform firmware describes the devices we will be able to check precisely if a given port exists or not in a device.